muddy_macros/src/internal.rs

```diff
@@ -42,7 +42,14 @@ pub(crate) fn build_obfuscation_mod(
                 Some(ref s) => s.as_str(),
                 _ => "MUDDY",
             };
-            eprintln!("{env_name}='{key}'");
+            eprintln!("Please, set {env_name} env variable with content:\n");
+            #[cfg(windows)]
+            // language=cmd
+            eprintln!(r#"set "{env_name}={key}""#);
+            #[cfg(not(windows))]
+            // language=sh
+            eprintln!(r#"{env_name}="{key}""#);
+            eprintln!();
```

Comment:
I like that this prints out better help for windows users, but this seems a bit much, how about something like this?:

```suggestion
            #[cfg(windows)]
            eprintln!(r#"set "{env_name}={key}""#);
            #[cfg(not(windows))]
            eprintln!(r#"{env_name}='{key}'"#);
```

Reply:
About `// language=cmd` and `// language=sh` comments — it's language injection comments in JetBrain's IDE, https://www.jetbrains.com/help/idea/using-language-injections.html#use-language-injection-comments used for syntax highlighting and adding context to code blocks.

![image](https://github.com/orph3usLyre/muddy-waters/assets/2821574/feb91d1a-c3eb-40cc-94c4-f0ef1129a945)

>  but this seems a bit much

It's used for remember what needs to be done.

How about to keep the information about what developer must to do?